packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -531,11 +531,11 @@ class Viewport {
 
   /**
    * Add a list of actors (actor entries) to the viewport
-   * @param resetCameraPanAndZoom - force reset pan and zoom of the camera,
+   * @param resetCamera - force reset pan and zoom of the camera,
    *        default value is false.
    * @param actors - An array of ActorEntry objects.
    */
-  public addActors(actors: ActorEntry[], resetCameraPanAndZoom = false): void {
+  public addActors(actors: ActorEntry[], resetCamera = false): void {
```

Comment:
Could you pass an options bag here - I think it only needs the one option, but it would be really nice to have the option of providing more values here.